kong/plugins/rate-limiting/policies/init.lua

```diff
@@ -2,17 +2,36 @@ local policy_cluster = require "kong.plugins.rate-limiting.policies.cluster"
 local timestamp = require "kong.tools.timestamp"
 local reports = require "kong.reports"
 local redis = require "resty.redis"
-
+local table_clear = require("table.clear")
```

Comment:
Here we should follow the above style, use:
`local tb_clear = require "table.clear"`